### ISSUE TITLE
MySQLInit() Port Attribute

### ISSUE THREAD

**Issue by fyroc** [GonistLelatel/xpdo#2]
Please consider adding a port attribute to the MySQLInit() function

**Comment by GonistLelatel** [comment]
ок

**Comment by GonistLelatel** [comment]
this "issue" can be resolved with inheritance the Database instance

```php
class DB extends Database 
{
	public function MySQLInit($user, $password, $dbname, $host = 'localhost', $port = '3306') 
	{
		$this->_pdo = new \PDO("mysql:host=$host;dbname=$dbname;charset=utf8mb4;port=" . $port, $user, $password);
		$this->_pdo->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
		$this->_isMYSQL = true;
	}
}
```

To resolve the singleton:
```php
$db = DB::getInstance();
\aphp\XPDO\Database::$instance = $db;

$db->MySQLInit($user, $password, $dbname, $host, $port);
```
Or to inheritance the \aphp\XPDO\Model::database(); static method
```php
class BaseModel extends \aphp\XPDO\Model {
  static function database() {
     return DB::getInstance();
  }
}
```

**Comment by GonistLelatel** [comment]
this is a minor issue, i won't do it at soon

**Comment by GonistLelatel** [comment]
**Fixed at 1.0.17**
